src/api/ResponseBuilder.ts

```diff
@@ -0,0 +1,81 @@
+import Discord, { Client, Message, TextChannel } from "discord.js";
+
+const WARN_COLOR = "#e3bb39";
+const FEEDBAP_COLOR = "#4b94e3";
+const SUCCESS_COLOR = "#33bd51";
+
+/**
+ * Send a warning message to the target channel.
+ */
+export const sendWarn = (response: string, message: Message) => {
+	const embed = new Discord.MessageEmbed().setColor(WARN_COLOR).setDescription(`⚠ ${response}`);
+	message.channel.send(embed);
+};
+
+/**
+ * Send a success message to the target channel.
+ */
+export const sendSuccess = (response: string, message: Message) => {
+	const embed = new Discord.MessageEmbed().setColor(SUCCESS_COLOR).setDescription(`✅ ${response}`);
+	message.channel.send(embed);
+};
+
+export const logFeedback = (client: Client, message: Message) => {
```

Comment:
For reference, you can access the client property of `Message` to get immediate access to the `Client` object.